services/datamanager/collection/collection.go

```diff
@@ -194,6 +196,12 @@ func GetCollections(params GetCollectionsParams) (results ReadCollections, err e
 			return
 		}
 
+		// Update count of remaining collections
+		remainingCollections = collections.ItemsAvailable - params.BatchSize - sdkParams["offset"].(int)
```

Comment:
Use `len(collections.Items)`, not `params.BatchSize` -- asking for `limit=100` doesn't guarantee we get 100 (even if we're not on  the last page).